scripts/build-glean-metadata.py

```diff
@@ -141,16 +164,17 @@ def get_app_variant_description(app):
         continue
 
     if not app_groups.get(app.app_name):
-        app_groups[app.app_name] = {
-            "app_name": app.app_name,
-            "app_description": app.app["app_description"],
-            "canonical_app_name": app.app["canonical_app_name"],
-            "deprecated": app.app.get("deprecated", False),
-            "url": app.app["url"],
-            "notification_emails": app.app["notification_emails"],
-            "app_ids": [],
-            "annotation": (annotations_index.get(app.app_name, {}).get("app")),
-        }
+        annotation = annotations_index.get(app.app_name, {}).get("app")
```

Comment:
Should we use `_get_annotation()` to get the annotation here too?

Reply:
App annotations currently have a slightly different structure in the annotations index (they are one level higher than for metrics pings), but I think we can rewrite the method to work for both. Will address before landing.